.github/workflows/ballerina-by-example.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Generate BBEs
-      run: ./ballerinaByExample/build-bbe.sh v1-2 bbes v1.2.2 true
+      run: ./ballerinaByExample/build-bbe.sh v1-1 bbes v1.1.4 true false
```

Comment:
Here the site version should be 1.1 isn't it? As we now changed the way we represent site versions

Reply:
having dot isn't an issue ?

Reply:
I checked already changed .me files available in 1.1/learn ... They seem to work